bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttBrokerConnection.java

```diff
@@ -577,6 +580,15 @@ public synchronized void start() throws MqttException, ConfigurationException {
     public synchronized void close() {
         logger.trace("Closing connection to broker '{}'", getName());
         try {
+            // Abort a connection attempt
+            isConnecting = false;
+
+            // Stop reconnect strategy from trying to reconnect
+            if (reconnectStrategy != null) {
+                reconnectStrategy.connectionEstablished();
```

Comment:
Why do you consider that a reconnect strategy implementation handles "close" and "connectionEstablished" the same way?
IMHO the reconnect strategy should be closed and leave it to the implementation if they act same on both methods.

Reply:
It should be the same for the strategy in regard that it should stop to reconnect. But if you see reasoning to split it, go for it.